### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by gforcada**
Minimal cleanup (with autopep8) plus the safe bits of 2to3.

### CODE REVIEW

**Comment by cjw296** [comment]
Hmm, can you figure out why Travis didn't build this?

File: mailinglogger/MailingLogger.py [hunk 1]
```diff
@@ -1,112 +1,112 @@
...
+            email['X-Log-Level'] = record.levelname
+            email['Date'] = formatdate()
+            email['Message-ID'] = make_msgid('MailingLogger')
+            smtp = smtplib.SMTP(self.mailhost, self.mailport)
+            if self.username and self.password:
+                smtp.login(self.username, self.password)
+            smtp.sendmail(self.fromaddr, self.toaddrs, email.as_string())
+            smtp.quit()
+        except:
+            self.handleError(record)
```
Comment (cjw296) [inline mailinglogger/MailingLogger.py]: Curious as to why GitHub has failed so badly at showing this diff!

Comment (gforcada) [inline mailinglogger/MailingLogger.py]: I think the problem was the line endings that was different

**Comment by gforcada** [comment]
I will try to see what's going on with travis...

**Comment by gforcada** [comment]
#10 seems to have triggered travis...

**Pull request closed without merging**